biorbd_optim/penalty.py

```diff
@@ -42,18 +43,33 @@ def minimize_markers(penalty_type, ocp, nlp, t, x, u, markers_idx=(), data_to_tr
                 penalty_type._add_to_penalty(ocp, nlp, val, **extra_param)
 
         @staticmethod
-        def minimize_markers_displacement(penalty_type, ocp, nlp, t, x, u, markers_idx=(), **extra_param):
+        def minimize_markers_displacement(penalty_type, ocp, nlp, t, x, u, rt_idx=-1, markers_idx=(), **extra_param):
```

Comment:
The purpose of minimize_markers_displacement is not to follow another marker, but two minimize the movement of which (i.e. minimize the velocity). However, it behaves better than minimize_marker_velocity. Is that what you really wanted?